translate/storage/statsdb.py

```diff
@@ -27,9 +27,9 @@
 import stat
 import sys
 from collections import UserDict
-import _thread
 from sqlite3 import dbapi2
 
+import _thread
```

Comment:
@nijel Is this not from the standard library?

Reply:
Not sure, will look into this

Reply:
I've rewritten the code to avoid using _thread, see https://github.com/translate/translate/pull/4057